modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Link.js

```diff
@@ -8,6 +8,17 @@ export default (options) => {
         linkOnPaste: true,
         HTMLAttributes: {}
       };
+    },
+    addAttributes() {
+      return {
+        ...this.parent?.(),
+        ...apos.modules['@apostrophecms/rich-text-widget'].linkSchema
+          .filter(field => field.htmlAttribute)
+          .reduce((obj, field) => {
+            obj[field.name] = { default: field.def ?? null };
```

Comment:
field.htmlAttribute is not a boolean, it is the attribute name to be used. Need to be able to pick up old values under that name and map them to the right schema field name and vice versa.